src/sumOfOther.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function sumOfOther(array) {
+  const arrOfSum = [];
+  for (let i = 0; i < array.length; i += 1) {
+    arrOfSum.push(array.filter(e => e !== array[i]).reduce((sum, current) => sum + current));
```

Comment:
[1,1,1,1] -> TypeError: Reduce of empty array with no initial value
[1,1,1,2] -> [2,2,2,3] but should be [4,4,4,3]

Method filter should check by index not by value.